### DISCUSSION TITLE
Windows Performance Mode Switching

### DISCUSSION THREAD

**Discussion by mirt81** [seerge/g-helper#1325] [Q&A]
Is there a option for builds after GHelper v0.109 to turn of Automatic Perfomance Mode switching ?

Th@nks !

**Comment by seerge** [comment]
@mirt81 "Performance" or "Power" Modes?

If you want to stick to only one Power mode (by some unknown reason), you can just manually set same one for all profiles under Fans+Power

**Reply by mirt81** [reply]
Th@nk You @seerge Giving attention !

Maybe it's just me. . . I don't want to use just one power mode. I no longer want to use ASUS' hacky power modes.

Im more use Windows Power modes  and switch them with a AHK script and in GHelper all the Time the Balanced Mode is set .
CPU Clocks are set in in the Windows Power plans Like 100%-80% for Performance or 99%-5% for Balanced plan . . . so after GHelper v0.109 there is the Issue if i have Performance Windows Power plan active and have run a Program that needs the power and that case allow not do something simultaneous on the G15 AE and i turn off the Screen by Screen 0ff Hotkey so after Screen 0n the Windows Power plan is switch automaticly to Balanced .

Since ASUS power states hacky I prefer to choose my own power states & draws as well as GHelper where possible : )

**Reply by seerge** [reply]
I have a feeling that you mix up Power plans and Power modes as well from what you wrote there.

Both can be customised and assigned in any order or combination you want already.

**Comment by Mi3sbbbi** [comment]
Sounds like this guy's needs are a little similar to mine,
